corehq/sql_proxy_accessors/sql_templates/get_reverse_indexed_cases.sql

```diff
@@ -1,6 +0,0 @@
-DROP FUNCTION IF EXISTS get_reverse_indexed_cases(TEXT, TEXT[]);
```

Comment:
This function will still exist on all environments which could be confusing. Might want another migration that just contains this line with a drop. (This processes is really annoying :/ )

Reply:
Oh you're right - I meant to add that in.  Will do so.